packages/notionable/src/global/account/holdings.ts

```diff
@@ -100,6 +100,7 @@ export function calculateHoldings(
       maturedTokenId,
       totalIncentiveEarnings,
       hasMatured: balance.hasMatured,
+      tokenType: undefined,
```

Comment:
Why is this in here?

Reply:
Fixes a type issue higher up. Doesn't hurt anything